### DISCUSSION TITLE
Budibase Installation Script

### DISCUSSION THREAD

**Discussion by rmutili** [Budibase/budibase#2153] [Ideas]
Hey,

Just thought if its possible to create a one file linux installation script which user only needs to download, change file to executable and install. This should install budibase on linux environment with few requirements prechecks. Just created one quick script and would love to hear your opinions.

[budibase installation script.zip](https://github.com/Budibase/budibase/files/6893443/budibase.installation.script.zip)

**Comment by mjashanks** [comment]
Hey Raphael. I hear your pain - there are a few steps involved in installing Budibase.

However, we have a few solutions on the horizon for different purposes:

**Cloud Sandbox**
If the purpose is simply to try/evaluate Budibase, we are currently working on our own "Cloud Sandbox", which will allow people to use Budibase online without installing. For now, this won't be for running in production - just to try it out. This should be ready by the end of September 2021.

**Hosting provider one-click deploys**
At the moment, Budibase is listed on the Digital Ocean ocean marketplace, which allows a one-click deployment onto Digital Ocean.

We will be adding to this over time for other providers, starting with [AWS](https://github.com/Budibase/budibase/discussions/986), [Azure](https://github.com/Budibase/budibase/discussions/987) and [GCP](https://github.com/Budibase/budibase/discussions/988).

**Kubernetes**
We will be working on a Kubernetes Helm Chart, which will allow cloud-agnostic deploys using Kubernetes: https://github.com/Budibase/budibase/discussions/984

If you have a preferred cloud provider, let us know by commenting under the discussions linked above or by creating a new one!

**Reply by rmutili** [reply]
Thanks Joe,

Looks interesting and promising. 

Sometimes new system engineers get lost in the wood trying to figure out the best approach when installing own on-premise production environment.

The budibase doc is awesome and allows for more approaches to installing budibase. Someday sometime, a one-click deployment on-premise will come in handy and it will help in launching a new instance on-premise.
